packages/sitecore-jss-nextjs/src/middleware/middleware.ts

```diff
@@ -55,13 +55,17 @@ export abstract class MiddlewareBase {
    */
   protected isPrefetch(req: NextRequest): boolean {
     const isMobile = req.headers.get('sec-ch-ua-mobile') === '?1';
+    const userAgent = req.headers.get('user-agent') || '';
+    const isKnownPlatform = /iPhone|Mac|Linux|Windows/i.test(userAgent);
+    const isKnownDevice = isMobile || isKnownPlatform;
```

Comment:
Should this also accound for browsers where sec-ch-ua-mobile is not supported?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Sec-CH-UA-Mobile#browser_compatibility

Reply:
Yes that was going to be my follow up question. Let's add Android, and Mac OS too.

Reply:
Android added.
Mac is already there.